frontend/src/app/components/LocalizedHtml.js

```diff
@@ -123,7 +123,7 @@ export default class LocalizedHtml extends Localized {
                 }
               );
             }
-            throw new Error('ftl string did not have as many anchors as the jsx');
+            throw new Error(`ftl string "${this.props.id}" did not have as many anchors as the jsx`);
```

Comment:
This is unrelated, but it helped me debug something while working on this branch, might as well get it into master.